### ISSUE TITLE
[Errno 8] Exec format error when commiting in version 4.0.0

### ISSUE THREAD

**Issue by morremeyer** [alessandrojcm/commitlint-pre-commit-hook#20]
# Description

When using (on Ubuntu 20.04)

* This repository, version 4.0.0
* pre-commit 2.9.2
* python 3.8.5
* git 2.25.1

with the [commitlint hook](https://github.com/alessandrojcm/commitlint-pre-commit-hook), an error occurs when using pre-commit as a hook (but not when calling it directly). 

# Error

When trying to commit, the following error occurs:

``` 
❯ git commit --all --message 'feat: test commit'
commitlint...............................................................Failed
- hook id: commitlint
- exit code: 1

[Errno 8] Exec format error: '/home/morre/.cache/pre-commit/repo6j0wed15/node_env-default/bin/commitlint'
```

However, running `pre-commit run --all-files` does not produce the same error.

Running the hook in version `v.3.0.0` also does not produce this error.

# Reproduction

## Steps
1. Create a new directory to test in and enter it
2. Initialize a git repository
3. Initialize a venv, activate it and install pre-commit==2.9.2
4. Add a hook for commitlint
4. Register pre-commit as commit-msg hook
5. Try to commit and see that it fails
7. Run pre-commit on all files directly and see that it works

## Copy-Paste

Execute

```sh
mkdir test
cd test
git init
python3.8 -m venv venv
source venv/bin/activate
pip install pre-commit
```

Create `.pre-commit-config.yaml` with:

```yaml
default_stages: [commit]
repos:
- repo: https://github.com/alessandrojcm/commitlint-pre-commit-hook
  rev: v4.0.0
  hooks:
  - id: commitlint
    stages: [commit-msg]
    additional_dependencies: ['@commitlint/config-conventional']
```

Execute

```sh
pre-commit install --hook-type commit-msg
git commit --all --message 'feat: test commit'
``` 

Note the error detailed above.

**Comment by alessandrojcm** [comment]
Hi! Thanks for reporting this; I'll take a look as soon as possible.

**Comment by alessandrojcm** [comment]
Sorry for the delay, I'm still trying to figure out this error. The error per see is strange, that error code is usually related to a missing shebang on a python script. But this one seems directly related to the `commitlint` CLI; since changing the hook's  entry for any other node code does not throw an error.

I'll keep looking into this.

**Comment by morremeyer** [comment]
Thanks for your work! If I can help out in any way, please let me know.

**Comment by alessandrojcm** [comment]
Ok, so taking another look this has me puzzled.

The `[Errno 8]` means that the Python interpreter tried to execute a script but the script was malformed, thus it fails. Usually is due to a missing shebang.

But, if we see the log, we see that the path is something like `'/tmp/tmp15_wnik9/repoy3spd2wb/node_env-default/bin/commitlint'`. So that sounds like `pre-commit` is trying to execute `commitlint` using the Python interpreter (?)

The only difference between version 3.0.0 and 4.0.0 of this repo is the `@commitlin/cli` version, but according to the [changelog](https://github.com/conventional-changelog/commitlint/blob/master/CHANGELOG.md) there were no breaking changes between said versions.

I though it could be a bug of `pre-commit`, but reverting to version 2.8.2 yield no different results. Any ideas?

**Comment by noxxious** [comment]
Since the commitlint is created as an executable, but does not have a shebang, it is executed with default binary interpreter,  if I edit the commitlint binary and add #!/usr/bin/env node then everything works fine.

**Comment by alessandrojcm** [comment]
Thanks for pointing that out, @noxxious, guess we can revert to 9.1.1 while the `commitlint` team sorts that issue out.

**Comment by morremeyer** [comment]
Wow, I didn’t get near this. Thanks for pointing us to it, @noxxious!

@alessandrojcm Will you take care of reverting to 9.1.1 or shall I open a PR?

**Comment by alessandrojcm** [comment]
I'm caught at work right no, so a PR would be great @morremeyer; thanks!

**Comment by alessandrojcm** [comment]
Just made the PR to revert the version folks, hopefully, the commitlint team will get this fixed soon. Thank you all very much for reporting!

**Comment by alessandrojcm** [comment]
This was fixed on #45, turns out I had the path to the CLI wrong all this time.
